runelite-client/src/main/java/net/runelite/client/ui/overlay/components/InfoBoxComponent.java

```diff
@@ -38,7 +38,6 @@
 @Setter
 public class InfoBoxComponent implements LayoutableRenderableEntity
 {
-	private static final int SEPARATOR = 3;
```

Comment:
This was fixed-size separator from bottom, why was it removed?

Reply:
I thought you came up with this because you didn't seem to use metrics.getdescent which needs to be used to get the base y coordinate for rendering text. I replaced it with metrics.getMaxDescent() so i figured it isn't needed anymore but i can understand if you want a defined seperator (it won't have to be 3 anymore though).

Reply:
No this was intentional to add fixed padding from bottom (as 2px is border size + 1 px is separator)